.travis.yml

```diff
@@ -11,7 +11,6 @@ install:
 env:
   - TOX_ENV=flake8
   - TOX_ENV=docs
-  - TOX_ENV=tests
```

Comment:
nevermind, I see you deleted the file in a later commit. You can actually nuke this commit entirely.